README.md

```diff
@@ -95,6 +95,27 @@ Install [jq](https://stedolan.github.io/jq) so that the parameters can get parse
 make test
 ```
 
+## Customize Logging
```

Comment:
I'm glad you added this doc. But can we split it into two sections, maybe `setting log level` and `customizing logging tags`?